tests/behaviour/config/parameters.py

```diff
@@ -16,20 +16,109 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+from datetime import datetime
+from enum import Enum
+from typing import List, Dict
 
-from typing import List
+import parse
+from behave import register_type
+from behave.model import Table
 
+from grakn.concept.type.value_type import ValueType
 from grakn.rpc.transaction import TransactionType
 
+# TODO: We aren't consistently using typed parameters in step implementations - we should be.
```

Comment:
worth making an issue?

Reply:
Yes - but there are many, many other issues like this. I was planning to compile the list once we are done with the stable release of Grakn 2.0.